db/queries.py

```diff
@@ -3,44 +3,67 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return [message for message in Message.objects.filter(text__contains=word)]
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return [chat for chat in Chat.objects.filter(title__startswith="Untitled")]
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+    return [message for message in Message.objects.filter(
+        sent__contains="2015"
+    ).values_list(
+        "user__first_name",
+        "user__last_name"
+    )]
 
 
 def get_actual_chats() -> list[Chat]:
-    pass
+    messages = Message.objects.filter(sent__contains="2020")
+    return [message.chat for message in messages]
 
 
 def get_messages_contain_authors_first_name() -> list[Message]:
-    pass
+    return [message for message in Message.objects.filter(
+        text__contains=F("user__first_name")
+    )]
 
 
 def get_users_who_sent_messages_starts_with_m_or_a() -> list[User]:
-    pass
+    return [
+        message.user for message in Message.objects.filter(
+            Q(text__istartswith="a") | Q(text__istartswith="m")
+        )]
```

Comment:
fix the break line here
u should write write message.user right after the [ - according to the closing quotes, or move the closing quotes to the next line